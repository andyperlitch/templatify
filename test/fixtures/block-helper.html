
<div class="helper-test">
  {{#list people}}{{firstName}} {{lastName}}{{/list}}
</div>
