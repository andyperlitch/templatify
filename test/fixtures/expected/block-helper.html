
<div class="helper-test">
  <ul>
    <li>Yehuda Katz</li>
    <li>Carl Lerche</li>
    <li>Alan Johnson</li>
  </ul>
</div>
